package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-is",
-  "version": "2.4.1",
+  "version": "3.0.0",
```

Comment:
Can you remove this change? I'll bump the version once I'll make the release :)

Reply:
Oh, sure!